src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1431,6 +1431,23 @@ public void stashNetworkModifications(UUID studyUuid, UUID nodeUuid, List<UUID>
         notificationService.emitElementUpdated(studyUuid, userId);
     }
 
+    @Transactional
+    public void updateNetworkModificationsActivation(UUID studyUuid, UUID nodeUuid, List<UUID> modificationsUuids, String userId, boolean active) {
+        List<UUID> childrenUuids = networkModificationTreeService.getChildren(nodeUuid);
+        notificationService.emitStartModificationEquipmentNotification(studyUuid, nodeUuid, childrenUuids, NotificationService.MODIFICATIONS_STASHING_IN_PROGRESS);
```

Comment:
is it intended to use MODIFICATIONS_STASHING_IN_PROGRESS? Don't we want a separate notification?

Reply:
Changed to MODIFICATIONS_UPDATING_IN_PROGRESS
I don't think we need a new type of notification